build.sh

```diff
@@ -348,6 +358,12 @@ if (( ${NUMARGS} == 0 )) || hasArg libraft || hasArg docs || hasArg tests || has
         echo "Building for *ALL* supported GPU architectures..."
     fi
 
+    # get the current count before the compile starts
+    if [[ "$BUILD_REPORT_INCL_CACHE_STATS" == "ON" && -x "$(command -v sccache)" ]]; then
```

Comment:
Can we raise some sort of warning if sccache is not available? Just thinking about a user using this who might not have it installed. We could even fail here just to make sure it's clear

Reply:
That's what this line does. If you don't have sccache it just goes ahead and collects the normal report without the cache hit ratio